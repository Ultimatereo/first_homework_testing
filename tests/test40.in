5
3 5 4 5 5 2
0 0 1 1 4 0
1 3 3 2 2 0
0 4 4 3 0 5
4 4 1 5 1 1
