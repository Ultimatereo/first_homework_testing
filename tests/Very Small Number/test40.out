-9.91666
13.0833
-11.3333
-0.666663
2.99999
